gnosis/safe-react
# Add Safe load error

## What it solves
Resolves #3025

## How this PR fixes it
If a Safe fails to load, determined by having `0` owners after the set polling interval (4.5s on prod., 15s on staging), an error screen is shown.

When clicking the "Back to Main Page" button, the error is reset and `lastViewedSafes` cleared for the current network.

## How to test it
1. Open an invalid `shortName:address` combination.
2. After the poll interval, observe the error page.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/148223457-698dbd50-d754-4ed2-bdfe-6d60659ce794.png)
